### PR TITLE
Enable unittest jobs for windows

### PR DESCRIPTION
**Pull request by peterjc123**

### CODE REVIEW

File: .circleci/unittest/scripts/environment.yml [hunk 1]
```diff
@@ -7,8 +7,11 @@ dependencies:
...
-  - librosa
+  - librosa #[not win]
   - pip
   - pip:
-    - kaldi-io
     - scipy
+    - kaldi-io #[win]
+    - PySoundFile #[win]
+    - librosa #[win]
+    - future #[win]
```
Comment (peterjc123) [inline .circleci/unittest/scripts/environment.yml:17]: Well, I don't know exactly this is needed. But without this, an error will be thrown.
```
Processing dependencies for torchaudio==0.6.0a0+b7648ac
Searching for future
Reading https://pypi.org/simple/future/
Downloading https://files.pythonhosted.org/packages/45/0b/38b06fd9b92dc2b68d58b75f900e97884c45bedd2ff83203d933cf5851c9/future-0.18.2.tar.gz#sha256=b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d
error: Download error for https://files.pythonhosted.org/packages/45/0b/38b06fd9b92dc2b68d58b75f900e97884c45bedd2ff83203d933cf5851c9/future-0.18.2.tar.gz#sha256=b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:852)
```

Comment (vincentqb) [reply .circleci/unittest/scripts/environment.yml:17]: Thanks for noting this

**Comment by vincentqb** [comment]
Great! Let me know when this is ready for review :)

The plan will be to merge #604 and close #493, right?

**Comment by mthrok** [comment]
@peterjc123 Could we completely separate windows and linux unites scripts?
It appears to me that mixing them makes it harder to maintain.

I can move the current scripts from `.circleci/unittest/scripts` to `.cicleci/unittest/scripts/linux`. And windows specific files should go to `.circleci/unittest/scripts/windows`.

**Comment by peterjc123** [comment]
@mthrok okay, I'll make the change if you move the existing script to that place.

**Comment by peterjc123** [comment]
> Great! Let me know when this is ready for review :)
> 
> The plan will be to merge #604 and close #493, right?

Yes, and It's ready for review but I'll consider @mthrok 's suggestion.

**Comment by mthrok** [comment]
The unit test scripts for linux are designed to work together with the underlying Docker image. This new Windows unit test does not follow the same structure, so it's better to separate Windows scripts and linux scripts for a long term maintainability.

**Comment by mthrok** [comment]
I'll do it a.s.a.p

**Comment by mthrok** [comment]
@peterjc123 #605 I moved all the linux unit test tools to `.circleci/unittest/linux`. (slight change from what originally suggested) Can you approve? then I can merge it. Also you can put all the necessary scripts under `.circleci/unittest/windows`.

**Comment by peterjc123** [comment]
@mthrok approved and will do that.

**Comment by vincentqb** [comment]
> Yes, and It's ready for review but I'll consider @mthrok 's suggestion.

sounds good!

nit: the PR is still marked both as draft and `[WIP]` :)

**Comment by peterjc123** [comment]
@vincentqb Marked as ready in both ways.

**Comment by vincentqb** [comment]
@peterjc123 -- do you want to also enable nightlies for windows?

**Comment by peterjc123** [comment]
@vincentqb Will do that later.

**Pull request merged**